Userland/Libraries/LibGUI/TableView.cpp

```diff
@@ -174,7 +174,8 @@ void TableView::keydown_event(KeyEvent& event)
     auto is_delete = event.key() == Key_Delete;
     auto is_backspace = event.key() == Key_Backspace;
     auto is_clear = is_delete || is_backspace;
-    if (is_editable() && edit_triggers() & EditTrigger::AnyKeyPressed && (event.code_point() != 0 || is_clear)) {
+    auto has_ctrl = event.modifiers() & KeyModifier::Mod_Ctrl;
```

Comment:
Now this isn't just about performing a clipboard action, so let's reword the commit to something appropriate :P